README.md

```diff
@@ -66,6 +66,27 @@ dxscanner [path]
 
 dxs [path]
 ```
+## Flags Usage 🏳️
+**Use flags to let DX Scanner do what you want!**
+
+Insert your auth code if your repo is private.
+```
+dxs -a=AUTH_CODE
+
+dxs [path] -a=AUTH_CODE
+```
+Insert impact on which dxs command should fail. See possible impacts below. If you use *small* impact dxs command fail also if you don't practice some practices with medium or high impact.
+```
+dxs --fail=PRACTICE_IMPACT
```

Comment:
Nobody knows what `PRACTICE_IMPACT` means. And it's not true because you can set `all` and it's not a practice impact.